src/target/riscv/riscv-013.c

```diff
@@ -3266,9 +3274,14 @@ static int write_memory_progbuf(struct target *target, target_addr_t address,
 		 * to be incremented if necessary. */
 
 		uint32_t abstractcs;
-		bool dmi_busy_encountered;
-		if (dmi_op(target, &abstractcs, &dmi_busy_encountered, DMI_OP_READ,
-					DMI_ABSTRACTCS, 0, false, true) != ERROR_OK)
+		result = dmi_op(target, &abstractcs, DMI_OP_READ, DMI_ABSTRACTCS,
+				0, false, true, false);
```

Comment:
Curious why retry is specified as false in this particular call, but then if the result comes back as busy, then a second read operation is made immediately, but with retry=true.  (Why not specify retry=true in the first call?  Is the first call variant more efficient, and usually expected not to have a busy result?)